src/ssl.c

```diff
@@ -3456,10 +3456,13 @@ int PemToDer(const unsigned char* buff, long longSz, int type,
 
     headerEnd += XSTRLEN(header);
 
+    if ((int)(headerEnd - (char*)buff) > sz - 2)
```

Comment:
Looks like this should be ">=", since reading from bufferEnd is out of bounds:

(headerEnd + 1) >= bufferEnd